pages/patterns/select-language/three-or-more-languages.md

```diff
@@ -123,7 +123,6 @@ Place the language selector menu at the top of the screen above site navigation,
 ## See pattern in use
 
 {:.usa-content-list}
-- [U.S. Department of Education](https://www.ed.gov/notices/english-la)
```

Comment:
**Issue (blocking):** Did you mean to remove this link? Look like it redirects to: https://www.ed.gov/about/ed-overview/notices-other-languages/persons-with-limited-english-proficiency

Reply:
Yes, this was removed because it no longer uses the "3 or more languages" pattern. See this [comment](https://github.com/uswds/uswds-site/pull/2995#discussion_r1869971041) for more discussion about it.